src/components/stage/stage.css

```diff
@@ -78,3 +81,11 @@
     pointer-events: none;
     overflow: hidden;
 }
+
+.dragging-sprite {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 1000; /* Above everything so it is draggable into other panes */
```

Comment:
It's always "above everything" until that next thing that wants to be. It would be nice to keep it low enough to not be totally arbitrary.  In scratch-blocks the category menu is 40 so I made the dropdowns use 50... so could you make this 60? Or maybe 100?